tgui/packages/tgui/interfaces/ClothingBooth/swatchForegroundShapes/diamond.tsx

```diff
@@ -0,0 +1,18 @@
+import { SwatchForegroundProps } from "../type";
+
+export const SwatchDiamond = (props: SwatchForegroundProps) => {
+  return (
+    <svg width="auto" height="100%" viewBox="0 0 6 6" xmlns="http://www.w3.org/2000/svg">
+      <path
+        transform="matrix(.64018 .76822 -.64018 .76822 -22.577782 0)"
+        d="M20.297247-19.378817h2.7552822v2.7552826H20.297247z"
+        fill={props.colour}
+      />
+      <path
+        d="M25.016985 2.4392049h.76603258v.76603401H25.016985z"
+        transform="translate(-22.577782)"
+        fill={props.colour}
+      />
+    </svg>
```

Comment:
No idea why this has two paths. It's a diamond, one path surely?

Reply:
The diamond has curves, take it up with the .svg optimiser not me 😰